src/check.c

```diff
@@ -40,19 +40,19 @@ t_err	ft_check_lines(char **lines, int *lsrc_c, int *total)
 
 t_type	ft_check_line_type(char *line)
 {
-	if (!ft_strncmp(line, "A ", 2))
+	if (*line == 'A')
```

Comment:
Shall we also accept lower case letters? Encountered some .rt files with a for ambient for example

Reply:
I'd say no: it is an identifier therefore case matters. 
Also the hittables all have two lowercase letters which makes for a cool distinction: Uppercase is stuff around the scene; lowercase is stuff in the scene (hittables/objects).

Reply:
If you feel strongly about it, we can keep it this way. For me it would have been just more convenient to be able to just copy the file and run it, instead of having to change the identifiers to upper case.

Had similar issues with other repos where the parse/check was so strict that it didn't accept files which would run under other progs. These were often also not very clear about what blocked them. Finally just discarded them as they were too user unfriendly.

Reply:
I think it just opens up some space for potential bugs (on the top of my head 'C' and 'cy' could be confused) for not much upside. 
On the flip side a search replace for a > A could make these files working

Reply:
Does not work for c > C tho. Would turn our cy into Cy. It remains a pain in the ass :(

Reply:
In VSCode: Find and replace (Ctrl+H) with Match Case (Alt+C) and Match Whole Word (Alt+W) should do the trick